lib/Pikirasa/RSA.php

```diff
@@ -25,6 +35,84 @@ public function fixKeyArgument($keyFile)
     }
 
 
+    /**
+     * Creates a new RSA key pair with the given key size
+     *
+     * @param null $keySize   RSA Key Size in bits
+     * @param bool $overwrite Overwrite existing key files
+     * @return bool Result of creation
+     *
+     * @throws Pikirasa\Exception
+     */
+    public function create($keySize = null, $overwrite = false)
+    {
+        $keySize = intval($keySize);
+        if ($keySize < self::MINIMUM_KEY_SIZE) {
+            $keySize = self::DEFAULT_KEY_SIZE;
+        }
```

Comment:
You have a condition here that is not covered with a test. Can you add one for this condition?

Reply:
Certainly!

Reply:
The existing tests already evaluate the `true` path, since `intval(null) === 0`, so I added one for the `false` path, by passing a key size of `4096`.

Also turned CodeCov reporting back on, since my fork is actually already connected to it (I stole the Travis config from another project, so I wasn't certain at the time), so we can see the actual code coverage clearly.

Reply:
See also [Travis](https://travis-ci.org/danhunsaker/pikirasa) and [CodeCov](https://codecov.io/gh/danhunsaker/pikirasa/branch/adds-create-method).